### ISSUE TITLE
Add support for format document and format selection

### ISSUE THREAD

**Issue by xoofx** [madskristensen/MarkdownEditor#12]
This needs to be first implemented in Markdig, see [issue 17](https://github.com/lunet-io/markdig/issues/17) for the wip.

After we have `Markdown.Nornalize()`, we should be able to plug this quite easily into the Format Document/Format selection. We may put formatting options in the settings

**Comment by madskristensen** [comment]
Yes!! Just let me know when the normalize method become available and I'll implement the commands

**Comment by madskristensen** [comment]
Any update on `Markdown.Normalize()`?

**Comment by xoofx** [comment]
No update. Still moving to a new home/location with my family, lots of non-computer related stuffs to do. Should be able to work back on markdig later in September.

**Comment by madskristensen** [comment]
This is fixed in the new v2 extension https://marketplace.visualstudio.com/items?itemName=MadsKristensen.MarkdownEditor2
